mkosi.build

```diff
@@ -49,18 +49,20 @@ if [ ! -f "$BUILDDIR"/build.ninja ] ; then
 
         init_path=$(realpath /sbin/init 2>/dev/null)
         if [ -z "$init_path" ] ; then
-            rootprefix=""
+                rootprefix=""
         else
-            rootprefix=${init_path%/lib/systemd/systemd}
-            rootprefix=/${rootprefix#/}
+                rootprefix=${init_path%/lib/systemd/systemd}
+                rootprefix=/${rootprefix#/}
         fi
 
         meson "$BUILDDIR" \
                 -D "sysvinit-path=$sysvinit_path" \
                 -D "rootprefix=$rootprefix" \
                 -D man=false \
                 -D translations=false \
-                -D version-tag="${VERSION_TAG}"
+                -D version-tag="${VERSION_TAG}" \
+                -D mode=developer \
+                -D b_sanitize=${SANITIZERS:-none}
```

Comment:
I think when `SANITIZERS` is set `ninja test` should be replaced with something like `meson test --timeout-multiplier=3`. Without that some tests time out every now and again. Also, it seems it would be somewhat more consistent if  `ASAN_OPTIONS` and `UBSAN_OPTIONS` were passed to the test suite as well.